metricbeat/module/docker/helper.go

```diff
@@ -16,13 +16,29 @@ type Container struct {
 	Socket *string
 }
 
-func InitCurrentContainer(container *docker.APIContainers) *Container {
+func (c *Container) ToMapStr() common.MapStr {
+	m := common.MapStr{
+		"id":   c.Id,
+		"name": c.Name,
+		// TODO: Is this really needed
+		"socket": GetSocket(),
```

Comment:
I'm trying to understand the use case behind it. It see that people will search for id and name, but will the socket information be of any use? I think this is useful in the `info` metricset, but it is required in the other metricsets. I'm hesitant to remove it as I'm sure there was a good reason to add it which I'm just not aware of yet.